src/include/catalog/column_catalog.h

```diff
@@ -58,6 +59,7 @@ class ColumnCatalogObject {
   uint32_t column_id;
   uint32_t column_offset;
   type::TypeId column_type;
+  size_t column_length;
```

Comment:
The same type question. `column_length` would be consistent with `column_offset` if also using `uint32_t`.